env/src/lib.rs

```diff
@@ -269,17 +269,25 @@ impl Builder {
         self
     }
 
+    pub fn try_init(&mut self) -> Result<(), SetLoggerError> {
+        log::try_set_logger(|max_level| {
+            let logger = self.build();
+            max_level.set(logger.filter());
+            Box::new(logger)
+        })
+    }
+
     /// Initializes the global logger with an env logger.
     ///
     /// This should be called early in the execution of a Rust program, and the
     /// global logger may only be initialized once. Future initialization
     /// attempts will return an error.
-    pub fn init(&mut self) -> Result<(), SetLoggerError> {
-        log::set_logger(|max_level| {
-            let logger = self.build();
-            max_level.set(logger.filter());
-            Box::new(logger)
-        })
+
+    pub fn init(&mut self) {
+        let logger = self.build();
+        let filter = max_level.set(logger.filter());
```

Comment:
`max_level` is cause of TravisCI failure, since it doesn't exist in this scope any more.

you can probably change to this-ish:
```
let logger = self.build();
let filter = logger.filter();
let logger = Box::new(logger); // <---- need to use let again to end previous binding's life, and begin another's
log::set_logger(logger, filter);
```

warning: I didn't compile that ^^^ so check my work 😄